classes/system/linux/system/repo/mos9.yml

```diff
@@ -10,10 +10,6 @@ parameters:
           source: "deb http://mirror.fuel-infra.org/mos-repos/ubuntu/9.0/ mos9.0-hotfix main restricted"
           architectures: amd64
           key_url: "http://mirror.fuel-infra.org/mos-repos/ubuntu/9.0/archive-mos9.0.key"
-        mirantis_openstack_proposed:
```

Comment:
If this isn't used already (as I understand mos9.0-updates == mos9.0-updates), this change should be in a separate (and preceding) commit.